### DISCUSSION TITLE
Duplicate the previous_accumulative_cost_override so I can compare 2 or more tariffs

### DISCUSSION THREAD

**Discussion by f1wade** [BottlecapDave/HomeAssistant-OctopusEnergy#414] [Q&A]
Hi 

I am looking to duplicate the cost override sensors, so that I can put in 2 or more in future, to then compare multiple different tariffs.

Is this easily doing from within Home Assistant?

Or is it something that would need to extend and implement within the Github project?

**Comment by BottlecapDave** [comment]
Hello. This is a feature that's part of the integration, so if you want to compare more than two tariffs, you'll need to update your version of the integration. I might change how these are setup in the future to support x overrides, but this is quite far down the list as bugs and wider features are higher up the list.
